common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/text/DaxTextInput.kt

```diff
@@ -201,15 +201,21 @@ class DaxTextInput @JvmOverloads constructor(
             if (isPasswordShown) {
                 hidePassword()
             } else {
-                showPassword()
+                showPassword(isEditing = binding.internalEditText.hasFocus())
             }
         }
     }
 
-    private fun showPassword() {
+    private fun showPassword(isEditing: Boolean) {
```

Comment:
You don't need the argument here. You have access to `binding.internalEditText.hasFocus()` directly within the method.